docs/receiving/verifying-payloads/how-manual.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Verifying Manually
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+The recommended way to verify webhooks is using our official libraries as outlined in the [How to Verify](./how.mdx) section.
```

Comment:
👨‍🎨  We might want to wrap this in an [admonition](https://docusaurus.io/docs/next/markdown-features/admonitions). 


```suggestion
:::note
The recommended way to verify webhooks is using our official libraries as outlined in the [How to Verify](./how.mdx) section.
:::
```

Reply:
Probably want to use `:::info` instead. Could you please update this?